source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,331 @@
+      :language:  csharp
+
+   .. output:: 
+      :language: json
+      :visible:
+
+      { "_id" : 3, "make" : "PRS", "models" : ["Silver Sky", "SE", "Custom"], "establishedYear" : 1985, "rating" : 9 }
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Array Operators
+---------------
+
+Array operators query data based on the value or quantity of elements in an array
+field. Array operators include ``All()``, ``Any()``, and ``Size()``.
+
+.. note:: 
+
+   The {+driver-long+} translates the ``Any()`` operator to the MongoDB ``$elemMatch``
+   query selector.
+
+   To learn more about the ``$elemMatch`` query selector, see
+   :manual:`$elemMatch </reference/operator/query/elemMatch/#mongodb-query-op.-elemMatch>`.
+
+For more information on the array operators, see the :manual:`Array
+Query Operators </reference/operator/query-array/>` page.
+
+The following example uses builders to create a query filter that matches all
+documents that have 3 elements in the ``models`` field.
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindSizeBuilder.cs
+      :language:  csharp
+
+   .. output::
+      :language: json
+      :visible:
+
+      { "_id" : 2, "make" : "Gibson", "models" : ["Les Paul", "SG", "Explorer"], "establishedYear" : 1902, "rating" : 8 }
+      { "_id" : 3, "make" : "PRS", "models" : ["Silver Sky", "SE", "Custom"], "establishedYear" : 1985, "rating" : 9 }
+      { "_id" : 4, "make" : "Kiesel", "models" : ["Ares", "Vader", "Solo"], "establishedYear" : 2015, "rating" : null }
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Element Operators
+-----------------
+
+Element operators query data based on the presence or type of a field.
+
+For a full list of element operators, see the :manual:`Element
+Query Operators </reference/operator/query-element/>` page.
+
+The following example uses builders to create a query filter that matches all
+documents that have a ``rating`` field.
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindExistsBuilder.cs
+      :language:  csharp
+
+   .. output::
+      :language: json
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+      { "_id" : 2, "make" : "Gibson", "models" : ["Les Paul", "SG", "Explorer"], "establishedYear" : 1902, "rating" : 8 }
+      { "_id" : 3, "make" : "PRS", "models" : ["Silver Sky", "SE", "Custom"], "establishedYear" : 1985, "rating" : 9 }
+      { "_id" : 5, "make" : "Ibanez", "models" : ["RG", "AZ"], "establishedYear" : 1957, "rating" : 7 }
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Evaluation Operators
+--------------------
+
+Evaluation operators analyze data on individual fields, or on the entire collection's
+documents. Common evaluation operators include ``Regex()`` and ``Text()``.
```

Comment:
Issue: I believe the evaluation operators are $regex and $text rather than the builder methods for these operators.

Suggestion: Clarify language around operators and builders where applicable. A potential solution could be to make a table that lists the operator and the corresponding builder when it makes sense.

Reply:
Changed the wording for the given examples